src/EdgeSplittingConn.cpp

```diff
@@ -129,6 +137,8 @@ bool wmtk::TetMesh::split_edge(const Tuple& loc0, std::vector<Tuple>& new_edges)
     }
 
     // new_edges
+    for (size_t t_id : n12_t_ids) m_tet_connectivity[t_id].set_version_number(m_tet_connectivity[t_id].timestamp + 1);
+    for (size_t t_id : new_t_ids) m_tet_connectivity[t_id].set_version_number(0);
```

Comment:
```suggestion
    for (size_t t_id : new_t_ids) m_tet_connectivity[t_id].set_version_number(m_tet_connectivity[t_id].timestamp + 1);
```

Reply:
do we need this?
I added this:
https://github.com/wildmeshing/wildmeshing-toolkit/blob/e9e53c70f22436940c811e886e7f2faf8d3eb2a5/src/EdgeSplittingConn.cpp#L63